Makefile

```diff
@@ -172,8 +176,12 @@ ci-bootstrap: .ci-bootstrap
 	@touch .ci-bootstrap
 
 # bootstrap the build by downloading additional tools that may be used by devs
+# Note: consider migrating away from tool dependencies
+# https://github.com/bufbuild/buf/issues/1233#issuecomment-1166754805
+# https://github.com/cockroachdb/cockroach/issues/67216#issuecomment-874176186
 bootstrap: ci-bootstrap
 	go generate -tags tools tools/tools.go
+	go install github.com/bufbuild/buf/cmd/buf@v1.25.0
```

Comment:
Note: consider migrating away from tool dependencies, it can cause runtime problems with tooling: https://github.com/bufbuild/buf/issues/1233#issuecomment-1166754805, 
https://github.com/cockroachdb/cockroach/issues/67216#issuecomment-874176186